spec/features/finance/payment_breakdown_spec.rb

```diff
@@ -205,69 +177,57 @@ def then_i_should_see_correct_breakdown_summary
 
   def then_i_should_see_the_correct_payment_summary
     within ".finance-panel__summary" do
-      expect(page).to have_content(number_to_pounds(total_payment_with_vat_breakdown))
+      expect(page).to have_content(number_to_pounds(total_payment_with_vat))
 
       expect(page).to have_content("Output payment")
-      expect(page).to have_content(number_with_delimiter(output_payment_total))
+      expect(page).to have_content(number_to_pounds(output_payment_total))
 
       expect(page).to have_content("Service fee")
       expect(page).to have_content(number_to_pounds(service_fee_total))
 
       expect(page).to have_content("VAT")
-      expect(page).to have_content(number_to_pounds(total_vat_breakdown))
+      expect(page).to have_content(number_to_pounds(total_vat))
     end
   end
 
   def then_i_should_see_the_correct_output_fees
     expect(page).to have_content("Output payments")
-    expect(page).to have_content(number_to_pounds(jan_starts_breakdowns[:output_payments][0][:per_participant]))
-    expect(page).to have_content(jan_starts_breakdowns[:output_payments][0][:participants])
-    expect(page).to have_content(number_to_pounds(jan_starts_breakdowns[:output_payments][0][:subtotal]))
+    expect(page).to have_content(number_to_pounds(jan_statement_calculator.additions_for_started))
+    expect(page).to have_content(jan_statement_calculator.started_count)
+    expect(page).to have_content(number_to_pounds(jan_statement_calculator.output_fee))
+
+    expect(page).to have_content(number_to_pounds(nov_statement_calculator.additions_for_started))
```

Comment:
I think we might only need to have January statement checks here, as November statement is selected later in the steps?